R/function_left_parentheses_linter.R

```diff
@@ -4,6 +4,29 @@
 #' (e.g. `mean  (1:3)`). Although this is syntactically valid, it makes the code
 #' difficult to read.
 #'
+#' @examples
+#' # will produce lints
+#' lint(
+#'   text = "mean (x)",
+#'   linters = function_left_parentheses_linter()
+#' )
+#'
+#' lint(
+#'   text = "stats::sd(c (x, y, z))",
+#'   linters = function_left_parentheses_linter()
+#' )
+#'
+#' # okay
```

Comment:
suggestion: other `okay` examples like an `if ()` statement where ` (` is allowed

Reply:
also an example with `function()` since rules are slightly different for signatures

Reply:
> also an example with `function()` since rules are slightly different for signatures

I have added an example, but not sure if this is what you had in mind. What is different for signatures?

Reply:
that's what i had in mind, just function calls vs. function signatures

Reply:
Ah, I see.